src/common/cursor.js

```diff
@@ -7,13 +7,13 @@
 
 import {Effects} from "reflex";
 
-/*::
+
 import type {Cursor} from "./cursor"
 export type {Cursor}
-*/
+
 
 export const cursor = /*::<from, to, in, out>*/
-  (config/*:Cursor*/)/*:(model:from, action:in) => [from, Effects<out>]*/ => {
+  (config:Cursor)/*:(model:from, action:in) => [from, Effects<out>]*/ => {
```

Comment:
Any reason to keep return type annotation as comment ? If it breaks babel or flow or something else we should report a bug.